portfolio/src/main/webapp/script.js

```diff
@@ -27,11 +27,16 @@ function addRandomGreeting() {
   greetingContainer.innerText = greeting;
 }
 
-function fetchName() {
+function getComments() {
   const messageContainer = document.getElementById('comments')
-  fetch('/data').then(response => response.json()).then(name => name.reduce((acc, curVal) =>
-    acc + "<p>" + curVal + "</p></br>")
-  ).then( (comments) => {
-    messageContainer.innerHTML = comments;
-  } );
+  fetch('/data').then( response => response.json()).then(comments => comments.reduce((acc, curVal) =>  {
+      return acc + "<p>" + curVal.comment + " - <b>" +  curVal.userName +"</b></p></br>"
+    }, "")
+    ).then( (comments) => messageContainer.innerHTML = comments);
+}
+
+function createListElement(text) {
+  const liElement = document.createElement('li');
+  liElement.innerText = text;
+  return liElement;
 }
```

Comment:
I don't believe this function is used anywhere. Please delete it and add re-add it in a PR that uses it (if you still need it somewhere).